### PR TITLE
Create .well-known/security.txt file

### PR DESCRIPTION
**Pull request by mauriciofauth**
Based on https://securitytxt.org/ standard.

### CODE REVIEW

File: .well-known/security.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Contact: mailto:security@phpmyadmin.net
```
Comment (mauriciofauth) [inline .well-known/security.txt]: I don't know if we keep `Contact: mailto:security@phpmyadmin.net`
or change it to `Contact: https://www.phpmyadmin.net/security/`.

Comment (williamdes) [reply .well-known/security.txt]: I would use the URL but the email seems more future proof

Comment (williamdes) [reply .well-known/security.txt]: But more likely to be scraped by bots for spamming ..

File: .well-known/security.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+Encryption: https://files.phpmyadmin.net/phpmyadmin.keyring
+Encryption: https://keybase.io/phpmyadmin_sec
+Canonical: https://www.phpmyadmin.net/.well-known/security.txt
+Policy: https://www.phpmyadmin.net/security/
```
Comment (williamdes) [inline .well-known/security.txt]:
```suggestion
Policy: https://www.phpmyadmin.net/security/
Preferred-Languages: en
```

**Comment by williamdes** [comment]
https://github.com/securitytxt/security-txt
https://tools.ietf.org/html/draft-foudil-securitytxt-06

**Pull request merged**